### PR TITLE
Add row for TLS server certificate verification changes

### PR DESCRIPTION
**Pull request by erik-anderson**

### CODE REVIEW

File: microsoft-edge/web-platform/site-impacting-changes.md [hunk 1]
```diff
@@ -48,7 +48,8 @@ This table lists:
...
+| Ignore modifications to `document.domain` by default | v109 | | The `document.domain` property historically could be set to relax Same-Origin-Policy and allow subdomains from a site to interact. This behavior will be disabled by default such that setting the `document.domain` property will have no effect.  For more information and workarounds, see [Microsoft Edge will disable modifying document.domain](/deployedge/edge-learnmore-origin-keyed-agent-cluster). |
+| New TLS server certificate verifier | v109 (unmanaged devices), v110 (managed devices) | | No site compatibility impacts are anticipated. If you have uncommon TLS server certificate deployments you should test in v109 to confirm there's no impact. For more information and testing guidance, see [Changes to Microsoft Edge browser TLS server certificate verification](/deployedge/microsoft-edge-security-cert-verification). |
```
Comment (mikehoffms) [inline microsoft-edge/web-platform/site-impacting-changes.md:52]:
```suggestion
| New TLS server certificate verifier | v109 (unmanaged devices), v110 (managed devices) | | No site compatibility impacts are anticipated. If you have uncommon TLS server certificate deployments, you should test in v109 to confirm there's no impact. For more information and testing guidance, see [Changes to Microsoft Edge browser TLS server certificate verification](/deployedge/microsoft-edge-security-cert-verification). |
```

Comment (mikehoffms) [reply microsoft-edge/web-platform/site-impacting-changes.md:52]: d/k why "if" and "and" are red.  I tried re-typing spaces and words, still red

File: microsoft-edge/web-platform/site-impacting-changes.md [hunk 2]
```diff
@@ -48,7 +48,8 @@ This table lists:
...
-| Ignore modifications to `document.domain` by default | v109 | | The `document.domain` property historically could be set to relax Same-Origin-Policy and allow subdomains from a site to interact. This behavior will be disabled by default such that setting the `document.domain` property will have no effect.  For more information and workarounds, see [Microsoft Edge will disable modifying document.domain](/deployedge/edge-learnmore-origin-keyed-agent-cluster).
+| Ignore modifications to `document.domain` by default | v109 | | The `document.domain` property historically could be set to relax Same-Origin-Policy and allow subdomains from a site to interact. This behavior will be disabled by default such that setting the `document.domain` property will have no effect.  For more information and workarounds, see [Microsoft Edge will disable modifying document.domain](/deployedge/edge-learnmore-origin-keyed-agent-cluster). |
```
Comment (mikehoffms) [inline microsoft-edge/web-platform/site-impacting-changes.md:51]:
```suggestion
| Ignore modifications to `document.domain` by default | v109 | | The `document.domain` property historically could be set to relax the same-origin policy and allow subdomains from a site to interact. This behavior will be disabled by default such that setting the `document.domain` property will have no effect.  For more information and workarounds, see [Microsoft Edge will disable modifying document.domain](/deployedge/edge-learnmore-origin-keyed-agent-cluster). |
```
I couldn't find the string "Same-Origin-Policy" in the linked article:
https://learn.microsoft.com/en-us/deployedge/edge-learnmore-origin-keyed-agent-cluster
or in the article it links to:
https://html.spec.whatwg.org/multipage/browsers.html#relaxing-the-same-origin-restriction

**Review by mikehoffms — CHANGES_REQUESTED** [review]
Finished a Writer review pass of entering Comments.

File: microsoft-edge/web-platform/site-impacting-changes.md (hunk 2 above)
Comment (erik-anderson) [reply microsoft-edge/web-platform/site-impacting-changes.md:51]: This line was changed because I added the trailing `|` to denote the end of the cell and I wasn't intending to get a review on its text. But I agree this is a good change, so taking it anyway!

**Review by mikehoffms — APPROVED** [review]

**Review by Reezaali — APPROVED** [review]
Approving based on the review done by @mikehoffms.
